packages/resource-deployment/scripts/role-assign-for-sp.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+set -eo pipefail
+
+# The script will create a new role assignment for a service principal
```

Comment:
it creates role and assign, please update comment. Thx

Reply:
The script does only assignment of a role as per https://docs.microsoft.com/en-us/cli/azure/role/assignment?view=azure-cli-latest#az-role-assignment-create